google-cloud-bigquery/src/main/java/com/google/cloud/bigquery/JobStatistics.java

```diff
@@ -614,6 +625,14 @@ public Long getNumDmlAffectedRows() {
       return numDmlAffectedRows;
     }
 
+    /**
+     * Detailed statistics for DML statements Present only for DML statements INSERT, UPDATE, DELETE
```

Comment:
Consider being less specific about which statements may yield stats?  There's at least two DML statement types missing here, MERGE and TRUNCATE TABLE.

Reply:
Okay -- reduced comment here.